dts/arm/st/g4/stm32g4.dtsi

```diff
@@ -546,7 +546,7 @@
 			interrupts = <94 0>;
 			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x4>;
 			dma-generators = <4>;
-			dma-requests= <21>;
+			dma-requests= <115>;
```

Comment:
The Ref man of the stm32g4 (RM0440) is  giving 115 in the DMAMUX instantiation table and counting the DMAMUX request inputs from 1 to 115 , (where inputs 1 to 4 are for generator). In the dmamux bindings, the dma-request is based on the stm32wb. There (RM0434),  the DMAMUX instantiation table gives 36 peripheral request inputs : the list is counting from 1 to 40 with 4 dmamux generators. It is confusing but I guess the dma-requests is 115-4=111 (as it should be in the G4 refman). Actually, the dma-request is the nb of peripheral request inputs (excluding the generator inputs).

Reply:
Thanks, fixed that now.